docs/dev/table/sourceSinks.md

```diff
@@ -791,7 +792,26 @@ tableEnv
   .inAppendMode()
   .createTemporaryTable("MySystemTable")
 {% endhighlight %}
+
 </div>
+
+<div data-lang="python" markdown="1">
+
+You can also use the Java TableFactory in Python. A default `CustomConnectorDescriptor` python class has been provided and you can use it in the API as follows:
```

Comment:
You can use a Java `TableFactory` from Python using the `CustomConnectorDescriptor`.